Sources/KlaviyoSwift/KlaviyoEnvironment.swift

```diff
@@ -13,14 +13,12 @@ var environment = KlaviyoEnvironment.production
 let PRODUCTION_HOST = "https://a.klaviyo.com"
 let encoder = {
     let encoder = JSONEncoder()
-    encoder.keyEncodingStrategy = .convertToSnakeCase
```

Comment:
🐍 case was biting me so I turned it off. Everything is manually specified now.

Reply:
mess with the snake you're gonna get the fangs